src/components/slider/Slider.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import RCSlider, { createSliderWithTooltip, SliderProps } from 'rc-slider';
+
+export interface ISliderProps {
+  slider?: SliderProps;
+  classes?: string[];
```

Comment:
Since we use `classnames` everywhere, I don't really see the point of classes properties being arrays. `className="foo foo2"` takes less space than `className={["foo", "foo2"]}`. Furthermore, these props could be named `className` to keep the same conventions as in React.

Reply:
We use classnames internally, but not necessarily externally. It might be shorter without, but I think the array is cleaner and is easier to interact with. Changing the prop name for `className` is a good idea though.